### DISCUSSION TITLE
Custom search engine and auto complete not working. Please help

### DISCUSSION THREAD

**Discussion by Bestboigill** [philc/vimium#4598] [General]
instructions in wiki isnt super clear i copies the regex and pasted in the options page but it isn't working

**Comment by philg-dev** [comment]
Could you please add more details on what you're trying to achieve?
What does RegEx have to do with custom search engines?
Which wiki page were you looking at? Please link it, so we might improve it for the future.

**Comment by Bestboigill** [comment]
what i want is to be able to prefix `y:` to search on youtube 
i have copied and pasted 
```
# Youtube.
y: http://www.youtube.com/results?search_query=%s Youtube
```
in the custom search engine box but it is not working , neither is the auto complete

**Comment by philg-dev** [comment]
The only thing "wrong" about your configuration that I can see is, that you're using `http` instead of `https`.

For YouTube there's already a stub configuration in the default Vimium configuration that is just commented out. So you would only need to remove the `#` at the start of the line to "activate" that as an additional search engine.

On top of that it might be worth noting that the custom Vimium search engines can only be used in the Vomnibar, so changes to those Vimium search engines will not affect the browser's native URL bar behavior.

Edit:
Here's the configuration that works fine for me:
```
y: https://www.youtube.com/results?search_query=%s Youtube
```

Another usage note: if you want to use the search engine prefix, you just need to type the letter(s) and hit `<SPACEBAR>`, do not use a `:` colon. So once you type `y ` (y followed by a space) the Vomnibar will "swallow" that prefix, make it appear as if the vomnibar is empty again and it will show "YouTube" below the input field to indicate the search engine that is being used.
